sirepo/template/srw.py

```diff
@@ -128,6 +129,7 @@
         labels=['Horizontal Position', 'Vertical Position', 'Intensity'],
         units=['m', 'm', '{intensity_units}'],
     ),
+    exportRsOpt=PKDict(filename=f'{_RSOPT_EXPORT_BASE_NAME}.zip',)
```

Comment:
This label is also _RSOPT_EXPORT_BASE_NAME

Reply:
[8ddc012](https://github.com/radiasoft/sirepo/pull/4057/commits/8ddc012d751d5458e94639decc0f52140fba1dec)